jenkins/Jenkinsfile

```diff
@@ -27,7 +27,10 @@ pipeline {
           }
           steps {
             sh '''
-              julia -e 'using Pkg; Pkg.Registry.update(); Pkg.develop(path="./NDTensors"); Pkg.develop(;path="."); Pkg.develop(; path="./ITensorGPU"); Pkg.add("CUDA"); using CUDA; CUDA.set_runtime_version!("v11.8"); Pkg.test("ITensorGPU");'
+              julia -e 'using Pkg; Pkg.add("CUDA"); using CUDA; CUDA.set_runtime_version!("v11.8"); Pkg.activate("./ITensorGPU"); Pkg.develop(path="."); Pkg.add(name="NDTensors", version="0.1.50"); Pkg.instantiate(); Pkg.test();' 
```

Comment:
Maybe we should use `Pkg.add(name="NDTensors", version="0.1")` instead of hardcoding to `"0.1.50"`, in case we update the version of NDTensors 0.1.